portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+    const facts =	
+    [
+        "I have been to over 30 states in the United States.", 
+        "I am the oldest of two children.",
+        "I can speak Spanish."
+    ];	
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  // Pick a random fact.	
+  const fact = facts[Math.floor(Math.random() * facts.length)];
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = "\n" + fact;
```

Comment:
You may not need this. If you want a space between "Click here.." and the message/fact, you should add this to the index.html page instead.